blueprints/case.py

```diff
@@ -0,0 +1,77 @@
+import json
+from flask import Blueprint, render_template, abort, jsonify, current_app, request, Response
+import pymongo
+from bson import ObjectId 
+from config.constants import DB_NAME
+import sys
+sys.path.append('.') # I DO NOT THING THIS IS A GOOD IDEA ON PRODUCTION CODE
+from data.repositories import Case_Repository
+
+case_api = Blueprint('case_api', __name__,
+                          template_folder='templates')
+
+client = pymongo.MongoClient()    
+db = client[DB_NAME]
+collection = db['case'] 
+case_repository = Case_Repository(collection, db)
+
+@case_api.route('/create_case', methods=['POST'])
+def create_case():
+  try:
```

Comment:
This variable name makes sense since it is the "response" provided by the user with the case information, but this could be confusing in this context because we are looking at the HTTP _request_ and the HTTP response is what we send back to the client. This name could be made more descriptive with a noun like "payload" , "request_body" or something even more specific like "case_data"